### PR TITLE
Add icon for zip file

### LINKED ISSUE #7: Add archive icons
Hello,

A quick question on how to contribute to this project.
_By the way, thank you for this extension :+1:._

I would like to add the atom icons for the archive _(ie: zip, tar.gz, etc...)_ and I don't know where to find them.
There is this one available on this website, but it seems different from the one used on atom: https://primer.style/octicons/file-zip-24

Also, what are the technical requirement for the svg file to respect the convention of this folder:
https://github.com/emroussel/atom-icons/tree/main/fileicons/images

Thank you in advance :slightly_smiling_face:

### PR DESCRIPTION
**Pull request by novln**
This will close #7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "atom-icons",
   "displayName": "Atom Icons",
   "description": "Atom file and folder icons for VS Code",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (emroussel) [inline package.json]: No need to increment the version in this PR, I'll do it when I make the release for this!

**Review by emroussel — CHANGES_REQUESTED** [review]
This looks great, just one comment

File: package.json (hunk 1 above)
Comment (novln) [reply package.json]: It's fixed.

**Review by emroussel — APPROVED** [review]
Thanks for the contribution! I'll make a release sometime in the next couple of days

**Comment by emroussel** [comment]
This is now published under 1.2.0

**Pull request merged**